### PR TITLE
Test with ruby 3.3

### PR DESCRIPTION
**Pull request by cjcolvar**
Part of https://github.com/samvera/maintenance/issues/159

### CODE REVIEW

**Review by jrochkind — APPROVED** [review]

**Comment by jrochkind** [comment]
oops, sorry this has a conflict now becuase of my other PR, but approved, and tag me after merge and I'll approve again if needed!

**Comment by cjcolvar** [comment]
@jrochkind Thanks for bumping this!  I did a quick merge so I hope it passes tests and looks good.

**Pull request merged**
